app/helpers/style_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module StyleHelper
+  def stylesheet_for_layout
+    if asset_exist?'custom.css'
+      'custom'
+    else
+      'application'
+    end
```

Comment:
I'm hoping that this helper will get more complicated in later versions (theme-list support? user setting?... etc.) so I think ternary operator may be a bit too much.